README.MD

```diff
@@ -96,16 +96,21 @@ options = {
 First of all you need to add a reference for your div:
 
 ```html
-<div WordCloud #word_cloud_chart=word-cloud [wordData]="word_cloud" [height]="300"></div>
+<div AgWordCloud
+          	#word_cloud_chart=ag-word-cloud
+          	[wordData]="word_cloud"
+          	 [options]="options">
+          </div>
 ```
 
 Update your component to this:
 
 ```.ts
+import { AgWordCloudData, AgWordCloudDirective } from 'angular4-word-cloud';
 export class MyComponent implements ..... {
 ...
-	@ViewChild('word_cloud_chart') word_cloud_chart: WordCloudDirective;
-	word_cloud: Array<WordCloudData> = [];
+	@ViewChild('word_cloud_chart') word_cloud_chart: AgWordCloudDirective;
+	word_cloud: Array<AgWordCloudData> = [];
 .....
 	updateWordCloud(wordCloud: WordCloud) {
```

Comment:
WorldCloud does not exist anymore as far as i can see. I just called the update method on the AgWorldCloudDirective after updating the cloud array.